packages/web-config-server/src/apiV1/dataBuilders/helpers/calculateOperationForAnalytics.js

```diff
@@ -0,0 +1,68 @@
+import { checkValueSatisfiesCondition } from '@tupaia/utils';
+import { NO_DATA_AVAILABLE } from '/apiV1/dataBuilders/constants';
+import { divideValues } from './divideValues';
+import { subtractValues } from './subtractValues';
+
+const checkCondition = (analytics, config) => {
+  const { dataElement, condition } = config;
+  const filteredAnalytics = analytics.filter(({ dataElement: de }) => de === dataElement);
+  if (filteredAnalytics.length > 1) {
+    throw new Error(`Too many results passed to checkConditions (calculateOperationForAnalytics)`);
+  } else if (filteredAnalytics.length === 0) {
+    return NO_DATA_AVAILABLE;
+  }
+  return checkValueSatisfiesCondition(filteredAnalytics[0].value, condition) ? 'Yes' : 'No';
+};
+
+const sumDataValues = (analytics, dataValues) => {
+  let sum; //Keep sum undefined so that if there's no data values then we can distinguish between No data and 0
+
+  analytics.forEach(({ dataElement, value }) => {
+    if (dataValues.includes(dataElement)) {
+      sum = (sum || 0) + (value || 0);
+    }
+  });
+
+  return sum;
+};
+
+const ARITHMETIC_OPERATORS = {
+  DIVIDE: divideValues,
+  SUBTRACT: subtractValues,
+};
+
+const performArithmeticOperation = (analytics, arithmeticConfig) => {
```

Comment:
This looks like it could be a candidate for `@beyondessential/arithmetic`! But out of scope for this PR.

Reply:
We can probably use indicators now!

Reply:
True! Should I add an indicator builder for CHECK_CONDITION? Probably would have been simpler, I just didn't think of it.

Reply:
We probably need to add support for an `analyticCount` builder (or similar), see the discussion starting in https://github.com/beyondessential/tupaia-backlog/issues/261#issuecomment-686868714.

It will take some time to properly implement this indicator. Until this happens, I suggest we keep your work here, and refactor those builders at a later stage, at which stage we should do an easy regression test